WordPress/Classes/ViewRelated/Feature Introduction/Blogging Prompts/BloggingPromptsFeatureIntroduction.swift

```diff
@@ -9,9 +9,8 @@ class BloggingPromptsFeatureIntroduction: FeatureIntroductionViewController {
     private var interactionType: BloggingPromptsFeatureIntroduction.InteractionType
 
     enum InteractionType {
-        // Two buttons are displayed, both perform an action. Shows a site picker
-        // if `blog` is `nil` and user has multiple sites.
-        case actionable(blog: Blog?)
+        // Two buttons are displayed, both perform an action.
+        case actionable(blog: Blog)
```

Comment:
"Shows a site picker" is no longer needed since the blog can't be `nil` and the `.informational` type doesn't show the "post" button. I'm not sure where there are so many entry points for this screen, but it looks like the one that required a site picker to be shown was removed at some point.

P.S. looks like another screen/flow that's an overkill